solr/contrib/ltr/src/test/org/apache/solr/ltr/TestLTRQParserPlugin.java

```diff
@@ -69,6 +69,20 @@ public void ltrModelIdDoesNotExistTest() throws Exception {
   }
 
   @Test
+  public void ltrBadRerankDocsTest() throws Exception {
+    final String solrQuery = "_query_:{!edismax qf='title' mm=100% v='bloomberg' tie=0.1}";
+    final SolrQuery query = new SolrQuery();
+    query.setQuery(solrQuery);
+    query.add("fl", "*, score");
+    query.add("rows", "4");
+    query.add("fv", "true");
+    query.add("rq", "{!ltr model=6029760550880411648 reRankDocs=-1}");
```

Comment:
potential suggestion for future similar tests: instead of specifically testing for `-1` you could here do something like `-random().nextInt(10)` to easily increase test coverage i.e. to test `-0..-9` range effectively.